contribute/design/1.x/csi/20190606-csi-volume-provisioning.md

```diff
@@ -457,6 +458,56 @@ spec:
 
 #### CStorVolumeReplica -- existing custom resource
 
+### CVC Controller Patterns
+These are some of the controller patterns that can be followed while implementing
+controllers.
+
+- Status.Phase of a CStorVolumeClaim (CVC) resource can have following:
+  - Pending
+  - Bound
+- Status.Phase is only set during creation
+  - Once the phase is Bound it should never revert to Pending
+- Status.Conditions will be used to track an on-going operation or sub status-es
+  - It will be an array
+  - It will have ConditionStatus as a field which can have below values:
+    - True
+    - False
+    - Unknown
+  - An item within a condition can be added, updated & deleted by the resource’s own controller
```

Comment:
in this case of CVC, its CSINode?

Reply:
It can be any CSINode. In other words the one which that receives the request.